server/.env

```diff
@@ -1,5 +1,7 @@
+REDIS_PREFIX=TOKEN_MINT:
+RPC_URL=http://127.0.0.1:8114/rpc
+API_PORT=3002
 SECP_CODE_HASH=0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8
 DEMO_PRIV_KEY_1=0xd00c06bfd800d27397002dca6fb0993d5ba6399b4238b2f29ee9deb97593d2bc
 DEMO_PRIV_KEY_2=0x63d86723e08f0f813a36ce6aa123bb2289d90680ae1e99d4de8cdb334553f24d
 PATH_TO_CONTRACTS=~/Nervos-dev/token_mint/server/deps/ckb-miscellaneous-scripts/build/
-PATH_TO_CKB_DB=/home/tannrallard/Nervos-dev/ckb-dev/data/db
```

Comment:
Would like to keep this so that the reset script still clears CKB db. A better approach would be to keep this and enable the reset script to take a flag for clearing CKB db or not

Reply:
I see resetting the CKB DB directly as being out of application scope and potentially introducing more problems than it's worth.

If you eventually want to support Testnet nodes (and potentially Mainnet nodes) the action becomes one where an unintended purge becomes more costly, timewise. Even for a dev node, this is a destructive action that only can be done under the assumption that the user does not have something on their dev node they wish to retain, such as data from another application they are working on.

For all node types, the node itself must be shut down before it is cleared. Failing to do so results in a panic by the node. In this regard, CKB is a distinctly different service that is not under the jurisdiction of Token Mint. The contrasting case being Redis, where Token Mint can properly clear its own data without requiring manual intervention or affecting anything else using the service. Since CKB will require manual action regardless, I believe it is better to train the user to do it using the proper method.